tests/d3d12_clear.c

```diff
@@ -716,6 +716,9 @@ void test_clear_unordered_access_view_image(void)
         {DXGI_FORMAT_R11G11B10_FLOAT, 1, 1, 0, 0, 1, 0, {{0}},
                 {0x3f000000 /* 1.0f */, 0 /* 0.0f */, 0xbf800000 /* -1.0f */, 0x3f000000 /* 1.0f */},
                 0x00000380, true},
+        {DXGI_FORMAT_B8G8R8A8_UNORM,  1, 1, 0, 0, 1, 0, {{0}},
+                {0, 0, 0x3f000080 /* 0.5f + epsilon */, 0x3f800000 /* 1.0f */}, 0xff000080, true},
+        {DXGI_FORMAT_B8G8R8A8_UNORM,  1, 1, 0, 0, 1, 0, {{0}}, {1, 2, 3, 4}, 0x04010203},
```

Comment:
Doesn't look like the code currently checks if uavs are supported for the format, which is an issues since bgra8 uavs are optional.

Reply:
Is there a single driver that doesn't support them?

Reply:
Old anv, intel on windows, turnip I think.

Reply:
I guess we should just add a format feature query before running the test.

Reply:
done.